javaslang/src/main/java/javaslang/collection/IndexedSeq.java

```diff
@@ -409,4 +452,24 @@ default boolean startsWith(Iterable<? extends T> that, int offset) {
             return -1;
         }
     }
+
+    interface Search {
+        static <T> int binarySearch(IndexedSeq<T> seq, Function1<Integer, Integer> comparison) {
```

Comment:
Good catch!

It's a little embarrassing: I quickly thought of this but couldn't find an adequate interface, moved on, and forgot – `IntUnaryOperator`, what a stupid name?!